packages/cysync-core/src/store/dialog/types.ts

```diff
@@ -123,6 +130,13 @@ export const GuidedFlowMap = {
   importWallet: 'importWallet',
 } as const;
 
+export const TroubleShootMap = {
```

Comment:
We don't need these types in TroubleShoot

Reply:
As of now I am using it to show `bydefault` once `Settings` UI is done will update this